src/styles/main.scss

```
@@ -1,3 +1,3 @@
-body {
-  margin: 0;
-}
+@import "./utils/variables.scss";
+@import "./blocks/calendar.scss";
+@import "./utils/normalize.scss";
```

```suggestion
@import "./utils/variables.scss";
@import "./utils/normalize.scss";

@import "./blocks/calendar.scss";
```
Import main blocks after dependencies. You can separate them by adding an empty line as well.